benchmark/VRFGasHelper.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
+pragma solidity >=0.5.3 <=0.6.4;
```

Comment:
Why is here the version from 0.5.3 to 0.6.4 and not fixed? I thought this option was used in libraries, but I'm actually not sure..

Reply:
you are right, we can live it as 0.6.4